src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -104,12 +104,11 @@ void Notifier::sendNotifyContextRequest(NotifyContextRequest* ncr, const std::st
     int          port;
     std::string  uriPath;
     std::string  protocol;
-    
+    std::string  initialUrl = url;
```

Comment:
url is passed to parseUrl() as const string& so... why an extra variable initialUrl is needed here?